ross/tests/test_materials.py

```diff
@@ -55,8 +57,10 @@ def test_specific_material(AISI4140):
 def test_error_rho():
     with pytest.raises(TypeError) as ex:
         Material(name="test", E=203.2e9, G_s=80e9)
-
-    assert "__init__() missing 1 required positional argument: 'rho'" == str(ex.value)
+    if sys.version_info.minor == 10:
+        assert "Material.__init__() missing 1 required positional argument: 'rho'" == str(ex.value)
+    else:
+        assert "__init__() missing 1 required positional argument: 'rho'" == str(ex.value)
```

Comment:
This solves the problem for version 10, but in the future we would need to add something for version 11...
Since the purpose of this test is just to check if the correct error message is being raised, I would suggest to just change from:

```python
assert "__init__() missing 1 required positional argument: 'rho'" == str(ex.value)
```

To:
```python
assert "__init__() missing 1 required positional argument: 'rho'" in str(ex.value)
```

Reply:
Great! I did not know that this was possible! I'll make this correction right now!